.buildkite/scripts/test_e2e.sh

```diff
@@ -44,7 +44,7 @@ ${WORKDIR}/go/oasis-test-runner/oasis-test-runner \
     --e2e.runtime.binary_dir ${WORKDIR}/target/${runtime_target}/debug \
     --e2e.runtime.loader ${WORKDIR}/target/default/debug/oasis-core-runtime-loader \
     --e2e.tee_hardware ${OASIS_TEE_HARDWARE:-""} \
-    --remote_signer.binary ${WORKDIR}/go/oasis-remote-signer/oasis-remote-signer \
+    --params.remote-signer/basic.binary ${WORKDIR}/go/oasis-remote-signer/oasis-remote-signer \
```

Comment:
Why change this parameter as it will likely be the same for all remote-signer scenarios? In a similar way, all the `e2e.*` are also scenario-specific parameters (shared by many scenarios).

Reply:
Recursive params are now implemented. Fixed.